### ISSUE TITLE
Feature: create-topology - Show help on incorrect arguments

### ISSUE THREAD

**Issue by bbros-dev** [ipspace/netsim-tools#28]
We ran into this early on. 

When hitting an error triggered by incorrect arguments, show the output from `create-topology --help`

**Comment by ipspace** [comment]
I think I understand what you'd like to see, but would still like to see an example or two as there are very few cases where I could see this being applicable (example: cannot open topology file). If you have a problem in your data model displaying CLI usage won't help you.

**Comment by bbros-dev** [comment]
> there are very few cases where I could see this being applicable

Agreed.  It is really nit-picking.  It is something I always appreciate as a (lazy!) casual user who doesn't carry various command, option and argument permutations in my head.

Nothing urgent.
